SplunkRumWorkspace/SplunkRum/SplunkRum.xcodeproj/project.pbxproj

```diff
@@ -511,7 +511,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```

Comment:
Is there a reason for changing the deployment target from 11 to 12?

Reply:
@seemk  for testing purpose i did that.